src/Compiler/CompilerOpenFPGA.cpp

```diff
@@ -789,6 +789,117 @@ bool CompilerOpenFPGA::DesignChanged(
   return result;
 }
 
+std::vector<std::string> CompilerOpenFPGA::GetCleanFiles(
+    Action action, const std::string& projectName,
+    const std::string& topModule) const {
+  std::vector<std::string> files;
+  switch (action) {
+    case Compiler::Action::Analyze:
+      files = {ANALYSIS_LOG, "port_info.json",
+               std::string{projectName + "_analyzer.cmd"}};
+      break;
+    case Compiler::Action::Synthesis:
+      files = {
+          std::string{projectName + "_post_synth.blif"},
+          std::string{projectName + "_post_synth.edif"},
+          std::string{projectName + "_post_synth.v"},
+          std::string{projectName + "_post_synth.vhd"},
+          std::string{projectName + ".ys"},
+          std::string{projectName + "_synth.log"},
+          SYNTHESIS_LOG,
+      };
+      break;
+    case Compiler::Action::Pack:
+      files = {
+          std::string{projectName + "_post_synth.net"},
+          std::string{projectName + "_pack.cmd"},
+          "check_rr_node_warnings.log",
+          "packing_pin_util.rpt",
+          "pre_pack.report_timing.setup.rpt",
+          std::string{projectName + "_openfpga.sdc"},
+          std::string{projectName + "_post_synth_ports.json"},
+          "vpr_stdout.log",
+          PACKING_LOG,
+      };
+      break;
+    case Compiler::Action::Detailed:
```

Comment:
Why Detailed ?
```suggestion
    case Compiler::Action::Placement:
```

Reply:
Here are the changes:
![image](https://user-images.githubusercontent.com/95262932/211931292-6241c500-1ab5-49c2-9e77-e7d5aaa88983.png)
Are you sure?

Reply:
Ok, let's leave as it is. We should change that to Placement later on. It's an artifact coming from old PR.
https://github.com/os-fpga/FOEDAG/pull/400/files